codec-socks/src/main/java/io/netty/handler/codec/socksx/v5/DefaultSocks5PrivateAuthRequest.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2023 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.handler.codec.socksx.v5;
+
+import io.netty.handler.codec.DecoderResult;
+import io.netty.util.internal.ObjectUtil;
+import io.netty.util.internal.StringUtil;
+
+/**
+ * The default {@link Socks5PrivateAuthRequest} implementation.
+ * <p>
+ * For custom private authentication protocols, this class can be extended to include
```

Comment:
This class is final so it can't be extended... Maybe you want to say you can implement the interface ?